### DISCUSSION TITLE
web interface: list all licenses that have a given policy

### DISCUSSION THREAD

**Discussion by nschloe** [keygen-sh/community#136] [Q&A] [answered]
I was thinking about weeding out unused policies, but I don't see a way of listing all licenses that belong to a particular policy. Right?

**Comment by ezekg** [comment, marked as answer]
The UI doesn't have a filtering UI yet, but you can use this URL format to filter by policy in the interim:

```
https://app.keygen.sh/licenses?policy=<id>
```
